Clinical-Genomics/cgweb
# validate reference genome

This PR should only allow reference genomes consisting of letters numbers dots and underscores 

this is a minor, since it is backwards compatible

Test:
1. install on stage
1. create micro order
1. test with letters numbers underscores dots and other characters and save in between

Expected result:
Only letters numbers underscores and dots should be accepted as reference name

- [x] review code @ingkebil 
- [x] tested by @patrikgrenfeldt and @sylvinite 
- [x] deploy @ingkebil
